### DISCUSSION TITLE
Dudas ejercicio 1 TP3

### DISCUSSION THREAD

**Discussion by matiaspalleres** [INGCOM-UNRN-P1/cursada-2023#100] [Preguntas y respuestas] [answered]
Consulta, en el ejercicio 1 la consigna pide un main que permita utilizar las funciones. Deberia ser un menu, no? 
Y en el ejercicio 1.3 Fusionadora de arreglos, los arreglos que se unen tendrian que estar cargados por la funcion cargador de arreglos?

**Comment by martinvilu** [comment, marked as answer]
> Consulta, en el ejercicio 1 la consigna pide un main que permita utilizar las funciones. Deberia ser un menu, no?

Como te quede cómodo

> Y en el ejercicio 1.3 Fusionadora de arreglos, los arreglos que se unen tendrian que estar cargados por la funcion cargador de arreglos?

Si
